info/src/main/res/values/strings.xml

```diff
@@ -17,4 +17,15 @@
     <!-- Since we heavily depend on the google maps api we are considering google map to handle the museum location search.-->
     <string name="museumGoogleMapSearchQuery" translatable="false"><![CDATA[https://www.google.com/maps/search/?api=1&query=The+Art+Institute+of+Chicago]]></string>
     <string name="versionInfo" translatable="false">Version %1$s</string>
+
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
+    <string name="signIn">Sign In</string>
+    <string name="homeZipCode">Home Zip Code</string>
+    <string name="memberId">Member ID</string>
+    <string name="changeInformation">Change Information</string>
```

Comment:
Move `changeInformation` resource up one line.

Reply:
I was thinking it would read easier if all the member-related strings were together in one block. Looking at this now....we only need this text for this screen, so order isn't as helpful as i thought at furst glance.